Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html

```diff
@@ -117,6 +117,12 @@ <h3 mat-dialog-title class="pb-color-primary" i18n="Display-notificationDialog|W
     <p i18n="Display-notificationDialog|Cookie notice@@WelcomeCookieNotice">
       This site uses cookies to ensure the basic functionality of the app.
     </p>
+    <div class="cookie-notices">
+      <p i18n="Display-notificationDialog|Annoyed by cookies notice@@PageInformationNewUrlNoCookiesText">
+        Annoyed by the start-dialogs? Klick here:
```

Comment:
```suggestion
        Annoyed by the start-dialogues? Click here:
```

Reply:
Maybe rephrase the whole sentence like "You deactivated cookies and keep getting those dialogues?"